sycl/plugins/unified_runtime/CMakeLists.txt

```
@@ -53,14 +53,16 @@ endif()
 if(SYCL_PI_UR_USE_FETCH_CONTENT)
   include(FetchContent)
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  # commit 036b9cfcd8fb4be9394449b5406e6402580a63c9
-  # Merge: 2ab0734 86f96f0
-  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
-  # Date:   Fri Oct 27 16:36:23 2023 +0100
-  #     Merge pull request #961 from hdelan/change-unions-to-stdvariant
-  #     [HIP][CUDA] Change unions in ur_mem_handle_t_ to stdvariant
-  set(UNIFIED_RUNTIME_TAG 036b9cfcd8fb4be9394449b5406e6402580a63c9)
+  set(UNIFIED_RUNTIME_REPO "https://github.com/bensuo/unified-runtime.git")
+
+  # commit 74f42f8fdd3b92b355c5769bd13d4d982e839c78
+  # Author: Ben Tracy <ben.tracy@codeplay.com>
+  # Date:   Fri Oct 6 12:57:03 2023 +0100
+  # [CUDA][EXP] CUDA adapter support for command buffers
+  # - Implement initial subset of command buffer functions for CUDA
+  # - Report command buffer extension as supported from device query
+  # - Some additional methods and refactoring to reuse existing adapter code
+  set(UNIFIED_RUNTIME_TAG 5ecaf66a2d0325cb9b8cfcb8e39a60810fa46cc2)
```

Now that https://github.com/oneapi-src/unified-runtime/pull/932 has merged please update `UNIFIED_RUNTIME_TAG` as follows and pull in the latest sycl branch changes. This should then be ready to take out of draft.

```suggestion
  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
  # commit be53fb3ba5bf1ac33051456d72c61b6a01d94a72
  # Merge: 55d432c 5ecaf66
  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
  # Date:   Tue Oct 31 13:22:52 2023 +0000
  #     Merge pull request #932 from Bensuo/cuda-cmd-buffers
  #     [CUDA][EXP] CUDA adapter support for command buffers
```